sdk/appconfiguration/app-configuration/test/internal/helpers.spec.ts

```diff
@@ -130,6 +139,40 @@ describe("helper methods", () => {
     });
   });
 
+  describe("serializeAsConfigurationSettingParam", () => {
+    [`[]`, "Hello World"].forEach((value) => {
+      it(`serializer doesn't throw on ${value} as feature flag value`, () => {
```

Comment:
Just to mention - there's no comments or anything explaining _why_ we do this. It's probably worth adding one in to explain why we're so lax about the checking here.

Reply:
Added one :)